server/src/events.mjs

```diff
@@ -92,10 +93,79 @@ function sendEventListenerAck(ws, oMsg) {
   logger.debug(`Sent event listener ack message: ${ackMessage}`);
 }
 
+// sendEvent function to handle pre- post- method/event triggers
+function sendEvent(ws, method, result, msg, fSuccess, fErr, fFatalErr) {
+  try {
+    if ( !isRegisteredEventListener(method) ) {
+      fErr.call(null, method);
```

Comment:
Should probably add a log statement that says that there is no registered event listener (Could not send ${method} event because no listener is active).

Reply:
We should have this log statement.

Reply:
logger added for non-registered events